crew

```diff
@@ -111,11 +110,13 @@ String.use_color = args["--color"] || !args["--no-color"]
 if @opt_verbose then
   @fileutils_verbose = true
   @verbose = 'v'
-  @mv_verbose = '-v'
+  @short_verbose = '-v'
+  @long_verbose = '--verbose'
 else
   @fileutils_verbose = false
   @verbose = ''
-  @mv_verbose = ''
+  @short_verbose = ''
+  @long_verbose = '--verbose'
```

Comment:
Should this be `@long_version = ''`?

Reply:
No, but @short_verbose should be equal to '-v'

Reply:
On second thought, can we just remove this variable?  It doesn't appear to be used anywhere.

Reply:
nevermind it is the way it's supposed to be. And no, `--long-verbose` since the argument is --verbose not --version

Reply:
I'm planning on going through all the package files at some point and making them compatible with crew's `-v` option, so we'll need it then. I'll take it out for now though.